core/src/utilities/helpers/navigation-helpers.js

```diff
@@ -1,13 +1,22 @@
 // Helper methods for 'navigation.js' file. They don't require any method from 'navigation.js` but are required by them.
-import { LuigiConfig } from '../../services/config';
+import {
+  LuigiConfig
+} from '../../services/config';
 import * as AuthHelpers from './auth-helpers';
 export const isNodeAccessPermitted = (
   nodeToCheckPermissionFor,
   parentNode,
   currentContext
 ) => {
-  if (LuigiConfig.isAuthorizationEnabled() && !AuthHelpers.isLoggedIn())
-    return false;
+  if (LuigiConfig.isAuthorizationEnabled()) {
+    const loggedIn = AuthHelpers.isLoggedIn();
+    const anon = nodeToCheckPermissionFor.anonymousAccess;
+
+    if (loggedIn && anon === "exclusive" ||
```

Comment:
What is going on with `true` and `"exclusive"` values for _anon_? Please provide some documentation to this setting.

Reply:
documentation is covered in a separate ticket https://github.com/kyma-project/luigi/issues/296